IntegrationTests/Services/AWSS3IntegrationTests/S3EventStreamTests.swift

```diff
@@ -0,0 +1,62 @@
+//
```

Comment:
An integration test that tests the event stream output x restXml combo using S3:SelectObjectContent API.